xsamtools/cram.py

```diff
@@ -22,6 +22,13 @@
 CramLocation = namedtuple("CramLocation", "chr alignment_start alignment_span offset slice_offset slice_size")
 log = logging.getLogger(__name__)
 
+def file_definition(fh):
+    return {
+        'cram': fh.read(4).decode('utf-8'),
+        'major_version': int.from_bytes(fh.read(1), byteorder='little'),  # order shouldn't matter with 1 byte
+        'minor_version': int.from_bytes(fh.read(1), byteorder='little'),  # but it's required
```

Comment:
byteorder is probably a required positional argument because `int.from_bytes` accepts iterables of unknown length (generators, for instance).

Reply:
I understand, just thought it was worth leaving a comment about.